js/TrickleButtonView.js

```diff
@@ -202,10 +203,30 @@ class TrickleButtonView extends ComponentView {
    */
   async continue() {
     const parent = this.model.getParent();
-    await controller.continue();
+    const childrenAdded = await controller.continue();
+    if (!childrenAdded) return;
+    await this.announceContentLoaded();
     await controller.scroll(parent);
   }
 
+  /**
+   * Announce a message to screenreaders letting them know that additional
+   * content has been loaded on the page.
+   */
+  async announceContentLoaded() {
+    const globals = Adapt.course.get('_globals');
+    const message = globals?._extensions?._trickle?.additionalContentLoaded;
+    if (!message) return;
+
+    notify.create({ _type: 'a11y-push', body: message });
+
+    // Rough estimate: 200ms per word + buffer
+    const words = message.split(' ').length;
+    const delay = (words * 200) + 300;
```

Comment:
Happy with this? Did it all work as expected? Does it need a wider device test?

Reply:
It works well except when using feedback popups. When you close the popup, the focus moves to the disabled Submit button and announces it. So, the "loading" message is spoken over and never heard.